typify/tests/schemas/various-enums.rs

```diff
@@ -863,7 +863,7 @@ impl<'de> ::serde::Deserialize<'de> for Ipv6Net {
 pub enum JankNames {
     Variant0(::std::string::String),
     Variant1(::std::collections::HashMap<::std::string::String, ::std::string::String>),
```

Comment:
These clash and so stay as Variant0 and Variant1. There is an argument to be made to call them `<INFERRED_NAME>0` and `<INFERRED_NAME>1` instead. It is fairly easy to add. Will let you make the call.

Reply:
This is surprising. Variants 0 and 1 have the same title; variants 1 and 2 have the same type (object).